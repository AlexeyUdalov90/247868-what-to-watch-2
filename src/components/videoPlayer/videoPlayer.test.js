import React from 'react';
import renderer from 'react-test-renderer';

import VideoPlayer from './videoPlayer.jsx';

it(`Video player correctly renders after relaunch`, () => {

  const tree = renderer.create(<VideoPlayer poster={``} video={``} isPlaying={false} />, {
    createNodeMock: (element) => {
      if (element.type === `video`) {
        return {
          current: {
            src: ``,
          },
        };
      }
      return null;
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
