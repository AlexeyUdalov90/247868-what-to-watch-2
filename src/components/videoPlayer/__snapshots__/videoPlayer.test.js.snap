// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video player correctly renders after relaunch 1`] = `
<video
  height="175"
  poster=""
  width="280"
/>
`;
